dev/faq/data-migration.md

```diff
@@ -22,3 +22,10 @@ Because DM has the feature of concurrently replicating data to the downstream in
 The `driver: bad connection` error indicates that anomalies have occurred in the connection between DM and the upstream TiDB database (such as network failure, TiDB restart and so on) and that the data of the current request has not yet been sent to TiDB at that moment.
 
 When this type of error occurs in the current version, use `stop-task` to stop the task and then use `start-task` to restart the task. The automatic retry mechanism of DM will be improved later.
+
+## What can I do when a replication task is interrupted with failing to get or parse binlog error `get binlog error ERROR 1236 (HY000), binlog checksum mismatch, data may be corrupted` etc. returned?
+
+This error may occur during the incremental replication process of the DM if the binlog file in the upstream exceeds 4 GB, and the DM encounters a replication interruption in the processing of the file (including interruption caused by abnormalities in common pause or stop tasks). It is because that the DM needs to store the replicated binlog position. However, MySQL officially uses unit32 to store it, so the offset value of binlog position of the part exceeding 4 GB overflows and a wrong binlog position is stored. After the task or DM-worker is restarted, this wrong binlog position is used to re-parse the binlog or relay log. In this case, you need to follow these steps to manually recover replication:
```

Comment:
```suggestion
During the DM incremental replication process, this error might occur if the binlog file in the upstream exceeds 4 GB and DM encounters a replication interruption when it processes this binlog file (including ordinary pause or stop tasks and abnormal interruptions). This is because DM needs to store the replicated binlog position. However, MySQL officially uses uint32 to store it, so the offset values of binlog positions that exceed 4 GB overflow and DM stores a wrong binlog position. After the task or DM-worker is restarted, this wrong binlog position is used to re-parse the binlog or relay log. Then the above error occurs. In this case, perform the following steps to manually recover replication:
```

Reply:
- may -> might (Because "may" has multiple meanings.)
- unit32 -> u**in**t32
- Avoid unnecessary "the"
- Try to make sentences simple and concise